src/pages/product/card/ProductCard.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react'
+import { useWishlist } from '../../wishlist/wishlist-context';
+import "../../../App.css";
+const ACTIONS = {
+  ADDTOWISHLIST : "add-to-wishlist", 
+  REMOVEFROMWISHLIST : "remove-from-wishlist"
+}
+function ProductCard({
+    _id,
+    title,
+    author,
+    price,
+    inStock,
+    rating,
+    productImage,
+    isFastDelivery,
+    categoryName
+})
+ {
+
+    const {wishlist_dispatch} = useWishlist();
+    function wishListHandler(){
+      wishlist_dispatch({type: ACTIONS.ADDTOWISHLIST, 
+      payload:{_id,title,author,inStock,rating,isFastDelivery,categoryName,productImage, price}});
```

Comment:
Don't pass these many items in the payload. 
Instead, you can pass the whole object. 
![image](https://user-images.githubusercontent.com/79859472/160060031-079fbce1-bbbc-40a6-8f8c-4ff296b3c507.png)

Reply:
Done all the changes👍🏻